python/src/deltachat/testplugin.py

```diff
@@ -209,241 +209,253 @@ def read_path(self, bn, mode="r"):
     return Data()
 
 
-@pytest.fixture
-def acfactory(pytestconfig, tmpdir, request, session_liveconfig, data):
+class ACFactory:
+    _finalizers: List[Callable[[], None]]
+    _accounts: List[Account]
+
+    def __init__(self, strict_tls, tmpdir, session_liveconfig, data) -> None:
+        self.strict_tls = strict_tls
+        self.tmpdir = tmpdir
+        self.session_liveconfig = session_liveconfig
+        self.data = data
```

Comment:
Note that this is the former AccountMaker class but de-indented as it does not run as a closure anymore but is explicitely instantiated -- Most of the class remained the same except the make_account() function.